packages/block-editor/src/components/url-popover/image-url-input-ui.js

```diff
@@ -369,6 +369,7 @@ const ImageURLInputUI = ( {
 											onSetLightbox( true );
 											stopEditLink();
 										} }
+										label={ __( 'Expand on click' ) }
```

Comment:
What's the reasoning behind this change? The label is exactly the same as the text in the menu item.

Reply:
I thought I would need this to access it by name, but realized while working on this PR that we don't need an explicit label to do that — this change has been reverted.